yt_dlp/extractor/vbox7.py

```diff
@@ -53,41 +60,33 @@ class Vbox7IE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        response = self._download_json(
-            'https://www.vbox7.com/ajax/video/nextvideo.php?vid=%s' % video_id,
-            video_id)
-
-        if 'error' in response:
-            raise ExtractorError(
-                '%s said: %s' % (self.IE_NAME, response['error']), expected=True)
-
-        video = response['options']
-
-        title = video['title']
-        video_url = video['src']
-
-        if '/na.mp4' in video_url:
-            self.raise_geo_restricted(countries=self._GEO_COUNTRIES)
-
-        uploader = video.get('uploader')
-
-        webpage = self._download_webpage(
-            'http://vbox7.com/play:%s' % video_id, video_id, fatal=None)
+        data = self._download_json(
+            'https://www.vbox7.com/aj/player/item/options', video_id,
+            query={'vid': video_id})['options']
 
-        info = {}
+        if '.mp4' in data['src']:
+            base_url = data['src'].rpartition('_')[0]
+        else:
+            base_url = data['src'].rpartition('.')[0]
```

Comment:
what do some `data['src']` values look like?

Reply:
This depends on the video. As far as I can tell for older videos:
`"https://media34.vbox7.com/sl/KWDFgB4GVvBbRXryw8afsw/1706738400/24/249bb972c2/249bb972c2_720.mp4"`
For newer videos:
`"https://edge349.vbox7.com/sl/SCfALacEV7f-Cbbs7iqbrQ/1706738400/3e/3e18bac5ad/3e18bac5ad.mpd"`
For removed/broken(?) videos:
`""`